includes/view/Shifts_view.php

```diff
@@ -152,12 +153,13 @@ function Shift_view($shift, ShiftType $shifttype, Room $room, $angeltypes_source
 
     $content = [msg()];
 
-    if ($shift_signup_state->getState() == ShiftSignupState::COLLIDES) {
-        $content[] = info(__('This shift collides with one of your shifts.'), true);
-    }
-
-    if ($shift_signup_state->getState() == ShiftSignupState::SIGNED_UP) {
-        $content[] = info(__('You are signed up for this shift.'), true);
+    $info = match ($shift_signup_state->getState()) {
+        ShiftSignupStatus::COLLIDES  => info(__('This shift collides with one of your shifts.'), true),
+        ShiftSignupStatus::SIGNED_UP => info(__('You are signed up for this shift.'), true),
+        default => null,
+    };
+    if (!is_null($info)) {
```

Comment:
Does this really makes the code more understandable? 

Before: Two easily understandable if conditions :white_check_mark: 
Now: a match, default case with null, if not null condition afterwards with assignment

`match` may be great in some cases. But here in my opinion it makes things more complicated (+ adds two lines :D ).

Reply:
we could also do this
```php
match ($shift_signup_state->getState()) {
    ShiftSignupStatus::COLLIDES  => $content[] = info(__('This shift collides with one of your shifts.'), true),
    ShiftSignupStatus::SIGNED_UP => $content[] = info(__('You are signed up for this shift.'), true),
    default => null, // do nothing
};
```

Reply:
@weeman1337 suggestion?

Reply:
Eventually just leaving it as is would be the easiest way here